src/Standards/PEAR/Tests/Files/IncludingFileUnitTest.inc

```diff
@@ -97,3 +97,6 @@ if ($var == false) {
 
 require_once ('blank.inc');
 include_once ('blank.inc');
+
+require_once( 'blank.inc' );
+include_once( 'blank.inc' );
```

Comment:
Maybe also add unit test case for:
```php
require_once ( 'blank.inc' );
include_once ( 'blank.inc' );
```
i.e. whitespace both before and after the parenthesis.

Reply:
This got me thinking more about this, and I'm wondering if modifying this sniff is really the way to go. It seems to be adding a lot of whitespace-related complexity to the fixer, when the sniff really has nothing to do with whitespace. There was already a precedent within the sniff, but checking on both sides of both parentheses is 6 different checks. It seems like the fixing should really be handled by sniffs specifically for whitespace, like `Squiz.WhiteSpace.LanguageConstructSpacing` and `Squiz.WhiteSpace.SemicolonSpacing`. The one whitespace check that was already part of this sniff makes sense, but I think maybe we should just add those two sniffs to the WordPres ruleset rather than adding more whitespace handling to this sniff.

If you agree @jrfnl, I'll close this PR, and we'll open one on WPCS instead to add those sniffs to the ruleset.

Reply:
@JDGrimes Good call. I concur with your analysis and would be happy for those two sniffs to be added to WPCS.